app/screens/send-bitcoin-screen/scanning-qrcode-screen.tsx

```diff
@@ -143,23 +147,36 @@ export const ScanningQRCodeScreen: React.FC = () => {
           })
           return
         }
-
-        Alert.alert(
-          LL.ScanningQRCodeScreen.invalidTitle(),
-          destination.invalidReason === "InvoiceExpired"
-            ? LL.ScanningQRCodeScreen.expiredContent({
-                found: data.toString(),
-              })
-            : LL.ScanningQRCodeScreen.invalidContent({
+        destination.invalidReason === "InvoiceExpired"
+          ? Alert.alert(
+              LL.ScanningQRCodeScreen.invalidTitle(),
+              LL.ScanningQRCodeScreen.expiredContent({
                 found: data.toString(),
               }),
-          [
-            {
-              text: LL.common.ok(),
-              onPress: () => setPending(false),
-            },
-          ],
-        )
+              [
+                {
+                  text: LL.common.ok(),
+                  onPress: () => setPending(false),
+                },
+              ],
+            )
+          : Alert.alert(
+              "Open Link",
+              `${data.toString()}\n\nAre you sure you want to open this link?`,
```

Comment:
our app is meant to be translatable, so we don't hard code string. you can have a look at `app/i18n/en/index.ts` and also `make codegen` to be able to update our translations with the new string

Reply:
Added "en" version of these strings